### PR TITLE
Introduced dump() helper function

### PR DESCRIPTION
**Pull request by mickaelandrieu**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | When selenium tests fails, we got not chance to figure out what happens. Now we got screenshots and dump of the HTML render if we use the `dump()` function. |
| Type? | improvement |
| Category? | TE |
| BC breaks? | no |
| Deprecations? | no |
| How to test? | in a mocha test, use `browser.dump()` then take a look to `errorDumps` folder: wow! |

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

File: tests/Selenium/commands/init.js [hunk 1]
```diff
@@ -40,6 +41,27 @@ module.exports = function initializePrestaShopBrowserCommands (browser) {
   });
 
   /**
+   * Generate a dump of page and display the url
+   */
+  browser.addCommand('dump', function dump () {
```
Comment (Shudrum) [inline tests/Selenium/commands/init.js]:
```javascript
function dump()
```

Please.

File: tests/Selenium/commands/init.js [hunk 2]
```diff
@@ -40,6 +41,27 @@ module.exports = function initializePrestaShopBrowserCommands (browser) {
...
+    return browser
+      .getUrl().then(function(url) {
+        console.log(url);
+      })
+      .getSource().then(function(source) {
+        var randomId = Math.floor((Math.random() * 1000) + 1);
+        var randomFilename = "error-"+randomId+".html";
+
+        fs.writeFile('./errorDumps/'+randomFilename, source, function(err) {
```
Comment (Shudrum) [inline tests/Selenium/commands/init.js]: Now we are using:

```javascript
`./errorDumps/${randomFilename}`
```

Idem for the `console.log` below please.

Comment (mickaelandrieu) [inline tests/Selenium/commands/init.js]: what do you mean by ``Idem for console.log`` ?

```
console.log(${url}); // realy ?
```

Comment (Shudrum) [inline tests/Selenium/commands/init.js]: Nope, the other one ^^

```javascript
console.log(`Generated HTML file available at: /errors/${randomFilename}`);
```

**Pull request merged**